docs/meshstack.configuration.md

```diff
@@ -234,6 +234,53 @@ In order to use GCD as a lookup provider you need to provide these credentials:
 
 The GCD Service User needs read access to the [GCD Directory API](https://developers.google.com/admin-sdk/directory/v1/get-start/getting-started).
 
+### Motto of the Day
```

Comment:
Isn't MOTD usually 'Message of the day' ? Or why did we go with Motto here

Reply:
Yeah that sounds better I have to admit